File: Famoser.MassPass.Api/src/Models/Request/Authorization/CreateAuthorizationRequest.php
<?php
/**
 * Created by PhpStorm.
 * User: famoser
 * Date: 28/05/2016
 * Time: 17:28
 */

namespace Famoser\MassPass\Models\Request\Authorization;


use Famoser\MassPass\Models\Request\Base\ApiRequest;

class CreateAuthorizationRequest extends ApiRequest
{
    public $AuthorisationCode;
    public $Content;
}

File: Famoser.MassPass.Data/Entities/Communications/Request/Authorization/CreateAuthorizationRequest.cs
using Famoser.MassPass.Data.Entities.Communications.Request.Base;

namespace Famoser.MassPass.Data.Entities.Communications.Request.Authorization
{
    public class CreateAuthorizationRequest : ApiRequest
    {
        public string AuthorisationCode { get; set; }
        public string Content { get; set; }
    }
}
